### PR TITLE
Final Touch 1.0

### PR DESCRIPTION
**Pull request by akos29**
### Final Touch 

In this request :

- update styling 
- add TV-Show Single Page feature 
- update code for further refinement

This will solve issue #29

### CODE REVIEW

**Review by krisbobo — APPROVED** [review]
# Status: Approved✅
Hi @akos29 
Great job cleaning up the code base and finishing up the code's style and navigation.
## Highlights:
- Linters✔️ 
- Links✔️
## Action:
- The project is complete. There's nothing else to say than that; it's time to merge it. 🎉

File: src/index.js [hunk 1]
```diff
@@ -1,10 +1,29 @@
...
-document.addEventListener('DOMContentLoaded', component);
\ No newline at end of file
+document.addEventListener('DOMContentLoaded', component);
+
+homeLinks.addEventListener('click', () => {
+  tvCardContainer.classList.add('hide');
+  tvLinks.classList.remove('active');
+  homeLinks.classList.add('active');
+  mainContainer.classList.remove('hide');
+  cardContainer.classList.remove('hide');
```
Comment (thecodechaser) [inline src/index.js:20]: - [x] The like functionality isn't consistent, Some times it increases one like and sometimes 4 likes. I would suggest you fix this issue.

File: src/index.js [hunk 2]
```diff
@@ -1,10 +1,29 @@
...
 
-document.addEventListener('DOMContentLoaded', component);
\ No newline at end of file
+document.addEventListener('DOMContentLoaded', component);
+
+homeLinks.addEventListener('click', () => {
+  tvCardContainer.classList.add('hide');
+  tvLinks.classList.remove('active');
+  homeLinks.classList.add('active');
+  mainContainer.classList.remove('hide');
```
Comment (thecodechaser) [inline src/index.js:19]: - [ ] When I'm trying to drop a comment on a movie, The comment model closes and the comment doesn't get added. I would suggest fixing this issue and the comment functionality should work as per the requirements.

**Review by thecodechaser — CHANGES_REQUESTED** [review]
## Hi @akos29, and @krisbobo👋, 

### You are doing a marvelous job on the JavaScript group capstone project and this is one of the best projects I have seen so far!
There are some issues that you still need to work on to prepare your project for the assessment but you are almost there!

image

## To Highlight 🎉
- No linter errors. :heavy_check_mark:
- Gitflow is followed. :heavy_check_mark:
- Professional README file. :heavy_check_mark:
- PR has a good title and summary. ✔️
- All tests are passed. ✔️



## Suggested changes

- [ ] Good job adding tests for `The number of items (home)`, I would suggest you also add tests for `The number of comments (comments popup)` as this is the project requirement.
- [x] I would suggest you place the test files in a separate directory, It will help users to identify code files and test files easily.
- [x] Check the comments under the review.

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- x
- y
- z

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: src/index.js (hunk 2 above)
Comment (akos29) [reply src/index.js:19]: We will work on it, thanks 🙏

File: src/index.js (hunk 1 above)
Comment (akos29) [reply src/index.js:20]: Thanks update accordingly

**Pull request merged**